src/main/java/com/programmers/VoucherManagementApplication/vo/Amount.java

```diff
@@ -0,0 +1,14 @@
+package com.programmers.VoucherManagementApplication.vo;
+
+public class Amount {
+
+    private final Long amount;
+
+    public Amount(long amount) {
+        this.amount = amount;
+    }
```

Comment:
```suggestion
	public static final int ZERO = 0;

    public Amount(long value) {
	    validGreaterThanZero(value);
        this.value = value;
    }

	private void validGreaterThanZero(long amount) {
		if (amount < ZERO) {
			throw new IllegalArgumentException(Message.INVALID_MINUS.getMessage());
		}
	}
```

Amount 라는 객체가 생성되려면 경계값이 존재할 것으로 예상됩니다.
CreateVoucherRequest에서 처리해야 할 validation이 있고, 도메인 객체인 Amount에서 처리해야할 validation은 각각 역할이 있을 것 같습니다.